mod/file/download.php

```diff
@@ -4,7 +4,12 @@
  * 
  * @package ElggFile
  */
-require_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
What's wrong with this pattern?
Putting an is_file() check in there will just create fatal errors further down if there is no file